pkg/instance/instance.go

```diff
@@ -16,6 +16,7 @@ import (
 	spdkapi "github.com/longhorn/longhorn-spdk-engine/pkg/api"
 	spdkclient "github.com/longhorn/longhorn-spdk-engine/pkg/client"
 
+	lhutils "github.com/longhorn/go-common-libs/utils"
```

Comment:
newline for separation

```suggestion
	lhutils "github.com/longhorn/go-common-libs/utils"
	
```

Reply:
Fixed, thanks